src/game_config_manager.cpp

```diff
@@ -75,15 +75,15 @@ game_config_manager::game_config_manager(const commandline_options& cmdline_opts
 	}
 
 	// Clean the cache of any old Wesnoth version's cache data
-	if(const std::string last_cleaned = prefs::get().last_cache_cleared_version(); !last_cleaned.empty()) {
+	if(const std::string last_cleaned = prefs::get()._last_cache_cleaned_ver(); !last_cleaned.empty()) {
```

Comment:
I get it's not a simple change, but ultimately I think this one should have the initial underscore removed…

Reply:
I can just rename it - the only side effect is it'll clear the cache the first time someone starts wesnoth after it's renamed.

Reply:
Actually, it shouldn't. If the new prefs key returns an empty string, all that will happen is the pref is set, and the next time they start wesnoth it will show that "last time cleaned" is equal to the current version, and therefore no cleaning will happen. Besides, cleaning doesn't purge the whole cache, only cache entries from old wesnoth versions.